build/zsh/files/system-zshrc

```diff
@@ -27,5 +27,6 @@
 #
 
 # Turn on the "new" completion system.  See zshcompsys(1).
+PROMPT='%n@%m:%~ $ '
```

Comment:
How about using the `%#` token so that it uses `#` for the root user and `%` for non-root.
and can we drop the extra space before that to make it consistent with the default bash prompt? 

(see below)

```
[Connected to zone 'r34' pts/2]
Last login: Sat May 23 14:20:24 2020 on pts/2
OmniOS 5.11     omnios-r151034-f57f507df0       May 2020
root@r34:~# zsh
root@r34:~ $
```

Please move it up one line so it doesn't split the comment from the line that goes with it
(or put the prompt right at the end of the script)